mongo/job_store.go

```diff
@@ -92,6 +96,45 @@ func (m *JobStore) CreateJob(ctx context.Context, id string) (job models.Job, er
 	return newJob, nil
 }
 
+// CreateTask creates a new task, for the given API and job ID, in the collection, and assigns default values to its attributes
+func (m *JobStore) CreateTask(ctx context.Context, jobID string, nameOfApi string, numDocuments int) (task models.Task, err error) {
```

Comment:
I don't think it's a good idea to have named return arguments  as well as returning explicit values, it should follow one pattern or the other. TBH it's questionable whether named return arguments are ever a good idea unless absolutely necessary.

Reply:
now removed names from here and from some other functions too